js/apps/admin-ui/src/groups/GroupTable.tsx

```diff
@@ -220,9 +220,7 @@ export const GroupTable = ({
           <ListEmptyState
             hasIcon={true}
             message={t(`noGroupsInThis${id ? "SubGroup" : "Realm"}`)}
-            instructions={t(
-              `noGroupsInThis${id ? "SubGroup" : "Realm"}Instructions`
-            )}
+            instructions=""
```

Comment:
```suggestion
            instructions={t("noGroupsInThisSubGroup")}
```
I guess you still want this text here, but also a text for when there are no groups:
```
"noGroupsInstructions": "You haven't created any groups. Create a group to get started.",
```

If so then you need to select that one based on id:

```suggestion
            instructions={t(id ? "noGroupsInThisSubGroup") : "noGroupsInstructions")}
```

Reply:
this way the group empty state is in sync with the others

Reply:
I left it blank intentionally.  The instructions seem redundant.

Reply:
> In that case the whole prop can be omitted no?

No, we can't omit the prop.  `ListEmptyState` is used all over the place.  We could make it optional, but that seems like a waste of time right now.

Reply:
Right, because we want a consistent UI, you can't just leave it blank.